boards/limifrog-v1/dist/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+source [find interface/stlink-v2.cfg]
+transport select hla_swd
+
+set WORKAREASIZE 0x4000
```

Comment:
Where does this value come from? I thought `0x4000 = 0d16384` which stands for 16 kB RAM (thats just what I thought), but the `stm32l1` hast 80kB of RAM.